getting-started/templates/systemlink-values.yaml

```diff
@@ -606,6 +606,9 @@ dataframeservice:
   kafka-ui:
     image:
       registry: *imageRegistryRef
+    imagePullSecrets:
+      # <ATTENTION> - Update this value when updating global imagePullSecrets
+      - name: niartifacts-secret
```

Comment:
That seems reasonable. niImagePullSecret doesn't necessarily need to be global since nobody will directly access the value.

Reply:
Updated. I did make it global because it felt weird to have it defined outside of the global scope (and away from the `imagePullSecrets` value, but will update if it's agreed we don't want it there.

Reply:
Right, it just has to go somewhere and might as well put it next to where it's primarily used.